packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -90,15 +90,15 @@
 <div class="DropdownWrapper">
   <button
     data-spirit-toggle="dropdown"
-    data-spirit-target="#dropdownDisabledAutoClose"
+    data-spirit-target="#dropdown-disabled-auto-close"
     class="Button Button--primary Button--medium"
     aria-expanded="false"
-    aria-controls="#dropdownDisabledAutoClose"
+    aria-controls="#dropdown-disabled-auto-close"
     data-spirit-autoclose="true"
```

Comment:
This doesn't work as expected, reported as https://jira.almacareer.tech/browse/DS-1157.